### DISCUSSION TITLE
Is Kratos ready for production?

### DISCUSSION THREAD

**Discussion by lukecyca** [ory/kratos#2386] [Q&A] [answered]
This looks like a really great project. The Ory website makes it sound like this is stable, battle-tested software that can be replied upon. However the releases are all marked "alpha" and "pre-release". I wasn't able to find any soft of release schedule or roadmap towards a 1.0 stable release.

**Comment by lukecyca** [comment, marked as answer]
After some more hunting, I found my answer here: https://www.ory.sh/docs/ecosystem/versioning

> Ory only releases software that is stable and ready for production!
The sandbox/incubating stage is an indicator of how much the API could change in the future, including backwards incompatible changes.

**Reply by iambudi** [reply]
Is this still incubating?
